README.md

```diff
@@ -26,6 +26,7 @@
     <li>⚡️ Kibana dashboard for searching and filtering data</li>
     <li>💎 Simple, Configurable and Powerful</li>
     <li>🔥 Built entirely using open-source tools</li>
+    <li>Test it to know it</li>
```

Comment:
Hey! Thank you for your contribution. But this line doesn't make sense to be added to the README.md

Would suggest to try and contribute to something which adds value to the open source project.